homeassistant/components/dynalite/__init__.py

```diff
@@ -1,24 +1,81 @@
 """Support for the Dynalite networks."""
-from dynalite_devices_lib import BRIDGE_CONFIG_SCHEMA
 import voluptuous as vol
 
 from homeassistant import config_entries
 from homeassistant.const import CONF_HOST
+from homeassistant.exceptions import ConfigEntryNotReady
 from homeassistant.helpers import config_validation as cv
 
 # Loading the config flow file will register the flow
 from .bridge import DynaliteBridge
-from .config_flow import configured_hosts
-from .const import CONF_BRIDGES, DATA_CONFIGS, DOMAIN, LOGGER
+from .const import (
+    CONF_ACTIVE,
+    CONF_AREA,
+    CONF_AUTO_DISCOVER,
+    CONF_BRIDGES,
+    CONF_CHANNEL,
+    CONF_DEFAULT,
+    CONF_FADE,
+    CONF_HIDDEN_ENTITY,
+    CONF_NAME,
+    CONF_POLLTIMER,
+    CONF_PORT,
+    DATA_CONFIGS,
+    DEFAULT_NAME,
+    DEFAULT_PORT,
+    DOMAIN,
+    LOGGER,
+)
+
+
+def num_string(value):
+    """Test if value is a string of digits, aka an integer."""
+    new_value = str(value)
+    if new_value.isdigit():
+        return new_value
+    raise vol.Invalid("Not a string with numbers")
+
+
+CHANNEL_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_NAME): cv.string,
+        vol.Optional(CONF_FADE): vol.Coerce(float),
+        vol.Optional(CONF_HIDDEN_ENTITY, default=False): vol.Coerce(bool),
```

Comment:
Please remove this option. The entity.hidden property is deprecated and should not be used.

Reply:
removed